src/parser/builder.rs

```diff
@@ -0,0 +1,77 @@
+use crate::{Error, UserAgentParser};
+
+// fn foo() {
+// /// ```rust
+// /// # use uaparser::*;
+// let _parser = UserAgentParser::builder()
+// .unicode(false)
+//     .user_agent()
+// /// ```
+//     }
```

Comment:
Oops, I think I left this in here erroneously.

Reply:
Updated with a `fixup` commit and rebased.